src/data/sources.json

```diff
@@ -2,7 +2,7 @@
   {
     "name": "The Guardian",
     "site": "https://www.theguardian.com/uk/money",
-    "base": "https://12ft.io/",
+    "base": "https://12ft.io/https://www.theguardian.com",
```

Comment:
As far as I could tell, this one was already behaving as expected. Did you get errors when you tried?

Reply:
Everything is working fine for `The Guardian`, please revert back your changes for this one